database/migrations/2018_08_15_102039_move_albums.php

```diff
@@ -1,58 +1,61 @@
 <?php
 
 use App\Album;
+use Illuminate\Database\Migrations\Migration;
 use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Facades\Schema;
-use Illuminate\Database\Schema\Blueprint;
-use Illuminate\Database\Migrations\Migration;
 
 class MoveAlbums extends Migration
 {
-    /**
-     * Run the migrations.
-     *
-     * @return void
-     */
-    public function up()
-    {
-    	if(count(Album::all()) == 0) {
-		    if (Schema::hasTable(env('DB_OLD_LYCHEE_PREFIX', '') . 'lychee_albums')) {
-			    $results = DB::table(env('DB_OLD_LYCHEE_PREFIX', '') . 'lychee_albums')->select('*')->get();
-			    foreach ($results as $result) {
-				    $album = new Album();
-				    $album->id = $result->id;
-				    $album->title = $result->title;
-				    $album->description = $result->description;
-				    $album->public = $result->public;
-				    $album->visible_hidden = $result->visible;
-				    $album->license = $result->license;
-				    $album->save();
-			    }
-		    } else {
-			    echo env('DB_OLD_LYCHEE_PREFIX', '') . "lychee_albums does not exists!\n";
-		    }
-	    }
-    	else
-	    {
-		    echo "albums is not empty.\n";
-	    }
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		if (count(Album::all()) == 0) {
+			if (Schema::hasTable(env('DB_OLD_LYCHEE_PREFIX', '').'lychee_albums')) {
+				$results = DB::table(env('DB_OLD_LYCHEE_PREFIX', '').'lychee_albums')->select('*')->get();
+				foreach ($results as $result) {
+					$album = new Album();
+					$album->id = $result->id;
+					$album->title = $result->title;
+					$album->description = $result->description;
+					$album->public = $result->public;
+					$album->visible_hidden = $result->visible;
+					$album->license = $result->license;
+					if ($result->takestamp != '') {
+						$album->created_at = date('Y-m-d H:i:s', substr($result->takestamp, 0, -4));
```

Comment:
I believe this should be `sysstamp`, not `takestamp`, and you should pass it directly, without `substr`, since it's in full seconds, not like `id` (see https://github.com/LycheeOrg/Lychee-Laravel/issues/89#issuecomment-467019143).

Reply:
you might be right. :D